### DISCUSSION TITLE
Unable to get sharing to work.

### DISCUSSION THREAD

**Discussion by rodmcdonald** [mcguirepr89/BirdNET-Pi#730] [Q&A] [answered]
I have followed the Sharing Guide on here and am still unable to access my pi on the web.

Here are the settings in my router:

![Router](https://user-images.githubusercontent.com/59549461/217456526-b2784dce-46df-414f-bd08-e9874c59bc3a.png)
 And my noip settings:
![Noip](https://user-images.githubusercontent.com/59549461/217456649-6cc5fd3f-7af2-4a19-a7fe-e1ac93c7d4e4.png)

And from my Caddyfile:

....................................Caddyfile..................................
--
http:// http://robbo-birdpi.hopto.org {

What have I missed?

**Comment by ehpersonal38** [comment]
Port forwarding is one of the most annoying things to troubleshoot. That said, I think I may have an idea. From a simple IP lookup it seems that your ISP is TPG Telecom Limited. A Google search leads me to this forum: https://community.tpg.com.au/t5/Broadband-Internet/Port-forwarding/td-p/101013 which states:

> Home Wireless Broadband uses CG-NAT. Carrier-grade NAT (Network Address Translation) is a type of network that assigns services with a private IP address, instead of a dynamic public IP address. Our network will then translate that private address into a public address. This means that the following items (which depend on internal NAT) will not work on Home Wireless Broadband:

> Port forwarding
Hosting web, email or file servers internally
Smart Home systems (e.g. accessing security camera footage remotely, home automation and printers)
Remote Access (i.e. accessing your home computer or devices from another location)

> If you need any of these items in your home setup, Home Wireless Broadband won’t be the right fit for you. Please [check your address on our website](https://www.tpg.com.au/) for one of our other great broadband plans.

**Comment by rodmcdonald** [comment, marked as answer]
Thanks. Although I am not on fixed wireless you have sent me on the right path. Running ShieldsUP from grc.com confirms that I have no ports open to the internet.  I will contact my isp for assistance.

**Reply by rodmcdonald** [reply]
Thanks. It turned out that my isp had all ports blocked. I am now online. Now to dig out my good microphone.
